commands/uploadphotos.js

```diff
@@ -0,0 +1,57 @@
+import { google } from "googleapis"
+import { authorize } from "../utilities/google-auth.js"
+import fs from "fs"
+import { ContextMenuCommandBuilder, ApplicationCommandType } from "discord.js"
+import axios from "axios"
+import path from "path"
+
+export const data = new ContextMenuCommandBuilder().setName("upload photos").setType(ApplicationCommandType.Message)
+
+export async function execute(interaction) {
```

Comment:
This seems a bit dangerous to allow anyone to run - I would recommend adding a check to only allow those with Administrator permission (basically the officers) as otherwise anyone could cause the bot to download arbitrary files into our disk and then into drive.